Dockerfile

```diff
@@ -28,11 +28,9 @@ ENV DEP_VERSION=0.5.0
 RUN wget https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -O /usr/local/bin/dep && \
     chmod +x /usr/local/bin/dep
 
-# Install gometalinter
-ENV GOMETALINTER_VERSION=2.0.12
-RUN curl -sLo- https://github.com/alecthomas/gometalinter/releases/download/v${GOMETALINTER_VERSION}/gometalinter-${GOMETALINTER_VERSION}-linux-amd64.tar.gz | \
-    tar -xzC "$GOPATH/bin" --exclude COPYING --exclude README.md --strip-components 1 -f- && \
-    ln -s $GOPATH/bin/gometalinter $GOPATH/bin/gometalinter.v2
+# Install golangci-lint
+ENV GOLANGCI_LINT_VERSION=1.16.0
```

Comment:
Can we keep gometalinter and just add golangci-lint for transition purposes? Otherwise all outstanding PRs will fail since metalinter will not exist. We can remove gometalinter shortly after.